pilot/pkg/model/context.go

```diff
@@ -394,6 +394,9 @@ const (
 
 	// IstioIngressNamespace is the namespace where Istio ingress controller is deployed
 	IstioIngressNamespace = "istio-system"
+
+	// IstioInboundInterceptionSplit is the key indicating whether proxy wants inbound/outbound listener split
+	IstioInboundInterceptionSplit = "INBOUND_INTERCEPTION_SPLIT"
```

Comment:
There is a separate place where we list all the node metadata. This is not the right place.

Reply:
And please use the feature name - this is a long and very confusing name. 

What we are trying to do is allow capture of all inbound ports - instead of only containerPorts. 
We have few other similar settings.

Reply:
@rshriram  I can't find a reasonable other place. Could you share some light? 
BTW: bootstrap_config.go might be. But I don't see any existing example. In the contrary, the above `istio-system` is used there but defined here.